### ISSUE TITLE
Final pixel polishes/tweaks for DC flavors

### ISSUE THREAD

**Issue by arouinfar** [phetsims/circuit-construction-kit-common#654]
During today's design meeting, I noticed that the "Add Realistic Bulbs" checkbox looks bigger than other checkboxes. I should review the DC flavors to see if there are any other pixel polishes we should take care of before going to RC.

**Comment by arouinfar** [comment]
@samreid I've finished reviewing DC and DC-VL in master, and I found a few small things. I reopened #562 because DC-VL still has the UI sounds that were turned off in DC. Additionally, let's make these changes:

- [ ] Reduce the size of the the "Add Realistic Bulbs" checkbox so it is the same size as the other checkboxes in the sim.
- [ ] Center the animation speed string in the play area between the carousel/control panels, like we did for the circuit element edit panels.
- [ ] Reduce tweaker increment of the fuse's Current Rating NumberControl to 0.1 so it is consistent with other components.
